src/components/Router.jsx

```diff
@@ -1,23 +1,35 @@
 import React from 'react';
 import { BrowserRouter, Switch, Route } from 'react-router-dom';
 import Navbar from './Navbar/Navbar';
-import OrderHistory from './OrderHistory/OrderHistory';
+import Orders from './Orders/Orders';
 import ProductDetails from './ProductDetails/ProductDetails';
 import Products from './Products/Products';
 import ShoppingCart from './ShoppingCart/ShoppingCart';
+import Filters from './Filters/Filters';
+import ProductAdd from './ProductAdd/ProductAdd';
+import OrderDetails from './OrderDetails/OrderDetails';
 
 export default function Router() {
   return (
     <BrowserRouter>
       <main className="container MainContainer">
         <Navbar />
         <Switch>
+          <Route path="/history:/:id">
```

Comment:
would it make sense to rename this path (and is there an extra colon after history) to `"/orders/:id"` ?

Reply:
Fixed here: 
https://github.com/dlimx/InDaHoodie/pull/7

Reply:
you're too fast :)